homework_4/03_event_settler_script/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"03_event_settler_script/models"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math/rand"
+	"net/http"
+	"time"
+)
+
+const host = "127.0.0.1"
+const betAPIport = "8787"
+const getBetsUrl = "/bets"
+const statusKey = "status"
+
+const eventUpdateUrl = "/event/update"
+const eventAPIport = "8080"
+
+func getBets() ([]models.BetResponseDto, error) {
+	url := fmt.Sprintf("http://%s:%s%s?%s=active", host, betAPIport, getBetsUrl, statusKey)
+
+	resp, err := http.Get(url)
+	if err != nil {
+		return []models.BetResponseDto{}, err
+	}
+
+	body, err := ioutil.ReadAll(resp.Body)
+	defer resp.Body.Close()
+	if err != nil {
+		return []models.BetResponseDto{}, err
+	}
+
+	var bets []models.BetResponseDto
+	err = json.Unmarshal(body, &bets)
+	if err != nil {
+		return []models.BetResponseDto{}, err
+	}
+
+	return bets, nil
+}
+
+func getUniqueSelectionIds(bets []models.BetResponseDto) map[string]bool {
+	set := make(map[string]bool)
```

Comment:
There is a nice approach where you can use an empty struct instead of a bool here.
This is not wrong by any means but empty struct are designed to be without value